packages-content-model/roosterjs-content-model-core/lib/coreApi/createEditorContext.ts

```diff
@@ -16,6 +16,7 @@ export const createEditorContext: CreateEditorContext = (core, saveIndex) => {
         allowCacheElement: true,
         domIndexer: saveIndex ? cache.domIndexer : undefined,
         zoomScale: domHelper.calculateZoomScale(),
+        rootDocumentFormat: {},
```

Comment:
Here, we can call getComputedStyle() to get the pixel value of root font size, put it into editor context. Then we don't need to calculate it anymore.

And we don't need to add a new format type here, just use a simple number or string value here is ok.

Reply:
For example

interface EditorContext {
  ...
  rootFontSize: getComputedStyle(....).fontSize
}

Reply:
Ok, we can do that. With the current code we would only get the computed style if we detect a rem style. But I guess with this it would be easier to read.

Reply:
I am going to set the rootFontSize as number so we can pass it in to the parseValueWithUnit.